prac_05/hex_colours.py

```diff
@@ -0,0 +1,35 @@
+"""
+Hex Colour Lookup
+Estimate: 20 minutes
+Actual: 22 minutes
+"""
+
+HEX_COLOURS = {
+    "aliceblue": "#f0f8ff",
+    "antiquewhite1": "#faebd7",
+    "aqua": "#00ffff",
+    "aquamarine": "#7fffd4",
+    "azure": "#f0ffff",
+    "beige": "#f5f5dc",
+    "bisque": "#ffe4c4",
+    "black": "#000000",
+    "BlanchedAlmond": "#ffebcd",
+    "blue": "#0000ff"
+}
+
+
+def main():
+    """Get color names until enter nothing."""
```

Comment:
"""Get color names from the user and display their hex codes.""" might be a better comment here. Yours is fine too.